js/example.js

```diff
@@ -13,6 +13,10 @@ module.exports = function (styleguide, filePath) {
   const buildPath = styleguide.path.build()
   let data = resolver.data(buildPath, filePath)
 
+  if (Object.keys(data).length === 0) {
```

Comment:
In the case that a developer has created a parseable JSON file and the file contents are `{}`. Maybe not a likely case? But, one that would kill the watch process otherwise.